### PR TITLE
update config docs

### PR DESCRIPTION
**Pull request by t4lz**

### CODE REVIEW

File: content/docs/reference/configuration.md [hunk 1]
```diff
@@ -120,9 +120,9 @@ configuration file containing all fields.
         "unix_streams": "bear.+"
       },
       "dns": false,
-    },
-    "copy_target": {
-      "scale_down": false
+      "copy_target": {
```
Comment (meowjesty) [inline content/docs/reference/configuration.md:123]: Wait, is this in the wrong place again?

Comment (t4lz) [reply content/docs/reference/configuration.md]: changed, but it should be fixed in the doc comments in the code.

Comment (meowjesty) [reply content/docs/reference/configuration.md]: Yeah, that's what weirded me out, I sent a PR for this today

https://github.com/metalbear-co/mirrord/commit/43229ee43f448aeb224bb2b8e268d65a42ff2060#diff-dea72eda9f0246025af946b83d775fed79e48ec094502e832d8063addda698eaR140

**Review by meowjesty — APPROVED** [review]
:+1:

File: content/docs/reference/configuration.md (hunk 1 above)
Comment (t4lz) [reply content/docs/reference/configuration.md]: I used the `configuration.md` file from `main`, but it wasn't updated in that PR.

**Pull request merged**